static/app/utils/theme/useDarkTheme.tsx

```diff
@@ -0,0 +1,27 @@
+import type {PropsWithChildren} from 'react';
```

Comment:
Lets add this to storybook, I really don't want anyone using this. We could instead do something like <ThemeProvider theme={darktheme}> where you can use theme.type to switch on the correct theme

Reply:
Good call, addressed in 74002616a6f